plugins/Channel/config.py

```diff
@@ -51,6 +51,9 @@ def configure(advanced):
     registry.Boolean(True, _("""Determines whether the output of 'nicks' will
     be sent in private. This prevents mass-highlights of a channel's users,
     accidental or on purpose.""")))
-
+conf.registerChannelValue(Channel, 'rejoinDelay',
+    registry.NonNegativeInteger(0, _("""Determines how many seconds the bot will wait
```

Comment:
I am not sure 0 is good default for this. WeeChat uses 30 for example and it's enough long time that people could think that the user could have written `/join #channel` by themselves and with the bot it could be thought that the bot admin gave the join command again.